src/tools/dev/scripts/bv_support/bv_ospray.sh

```diff
@@ -314,6 +318,22 @@ function build_ospray
         if [[ ! -d $OSPRAY_BUILD_DIR ]] ; then
             echo "Making build directory $OSPRAY_BUILD_DIR"
             mkdir $OSPRAY_BUILD_DIR
+            if [[ -f $OSPRAY_LIBS_FILE ]] ; then
+	        tar zxf $OSPRAY_LIBS_FILE
+	        mkdir $OSPRAY_BUILD_DIR/ispc
+	        mkdir $OSPRAY_BUILD_DIR/ispc/src
```

Comment:
`mkdir -p`?

Reply:
Didn't know about that option. Thanks for sharing!